running `make dist PLATFORM=linux PYVERSION=py310`, got python: not found

building the package blenderbim locally on linux https://blenderbim.org/docs/devs/installation.html#bundling-for-blender
Got Error: 
```bash
python scripts/bbim_translations.py -i "dist/working" -o "dist/blenderbim"
/bin/sh: 1: python: not found
make: *** [Makefile:547: dist] Error 127
```
`python` should be replaced to `$(PYTHON)` here
https://github.com/IfcOpenShell/IfcOpenShell/blob/48d6cc8a286f076e9f75abbdab8ee7b25d704115/src/blenderbim/Makefile#L701

--- a/src/blenderbim/Makefile
+++ b/src/blenderbim/Makefile
@@ -698,7 +698,7 @@ endif
 
 	# Generate translations module for BBIM build
 	git clone https://github.com/IfcOpenShell/blenderbim-translations.git dist/working
-	python scripts/bbim_translations.py -i "dist/working" -o "dist/blenderbim"
+	$(PYTHON) scripts/bbim_translations.py -i "dist/working" -o "dist/blenderbim"
 	rm -rf dist/working
 
 	# Remove dependencies also bundled with Blender

